index.js

```diff
@@ -15,7 +15,7 @@ var projectDir = process.cwd();
 var Promise = require('promise');
 var URL = require('url');
 var readlineSync = require('readline-sync');
-var regexParseProjectName = /(.+:\/\/.+?\/|.+:)(.+\/.+)+.git/;
+var regexParseProjectName = /^([^:]+:\/\/[^\/]+?\/|[^:]+:)([^\/]+\/[^\/]+?)(?:\.git)?\s*$/;
```

Comment:
Very late comment. Do you mind explaining the regular expression so I dont have to do that work myself. Possibly with exampled.

Reply:
I’m not sure what to say: I believe I explained it sufficiently in the PR description, though I’ll clarify one last piece in saying that the robustness improvements I speak of are replacing `.` with <code>[^<em>x</em>]</code>, for whatever <code><em>x</em></code> is supposed to follow it. Beyond those things, it’s still the same regular expression as was there before.